### PR TITLE
build and binary name updates

### PR DESCRIPTION
**Pull request by sezero**

### CODE REVIEW

**Comment by psi29a** [comment]
I think it's going to take more than that I think...

**Comment by sezero** [comment]
CMake'ry needs updating yes, and really fixing.

Should I copy over wildmidi cmake'ry here?

**Comment by psi29a** [comment]
If you believe it will fix it, please do. 

Honestly, why do we have vproj and sln files? Shouldn't cmake be doing this for us anyway? I would really like to get rid of all these IDE specific things and focus on "making cmake great again". ;)

**Comment by sezero** [comment]
> If you believe it will fix it, please do.

OK, will try

> Honestly, why do we have vproj and sln files? Shouldn't cmake
> be doing this for us anyway?

Because I detest cmake :)
The makefiles and project files are more direct, and our cmake'ry
is broken anyway.

> I would really like to get rid of all these IDE specific things and
> focus on "making cmake great again". ;)

Will try poking the cmakefile

**Comment by sezero** [comment]
The appveyor thing doesn't fail now. I also tried myself using MSVC and it succeeds:
it automatically builds unsf.dll, unsf.exe, unsf_static.lib and unsf-static.exe

File: unsf.pc.in [hunk 1]
```diff
@@ -1,11 +1,15 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+# WildMIDI pkg-config file
```
Comment (psi29a) [inline unsf.pc.in]: Missed one

File: unsf.pc.in [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+includedir=${exec_prefix}/include
 
 Name: UnSF
 Description: UnSF converts soundfonts into gus patch files
-Requires: @PKG_CONFIG_REQUIRES@
-Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -l@LIBNAME@ @PKG_CONFIG_LIBS@
-Cflags: -I${includedir} @PKG_CONFIG_CFLAGS@
+Version: @UNSF_VERSION@
+URL: https://www.mindwerks.net/projects/wildmidi/
```
Comment (psi29a) [inline unsf.pc.in]: can be updated to https://www.mindwerks.net/projects/unsf/

I create the project page

**Comment by psi29a** [comment]
The rest looks good. :)

File: unsf.pc.in (hunk 1 above)
Comment (sezero) [reply unsf.pc.in]: Removed it now

File: unsf.pc.in (hunk 2 above)
Comment (sezero) [reply unsf.pc.in]: Updated the URL now

**Comment by psi29a** [comment]
Great! Thanks!

**Pull request merged**